module_tunnels/koupleless_http_tunnel/http_tunnel.go

```
@@ -287,7 +289,7 @@ func (h *HttpTunnel) QueryAllBizStatusData(ctx context.Context, nodeName string)
 	defer func() {
 		h.queryAllBizLock.Unlock()
 		if h.queryAllBizDataOutdated {
-			go h.QueryAllBizStatusData(ctx, nodeName)
+			go h.QueryAllBizStatusData(nodeName)
```

golangci-lint (1.62.2)

Error return value of `h.QueryAllBizStatusData` is not checked

(errcheck)